### ISSUE TITLE
Error creating experiment for auto-monarch

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#801]
In GitLab by @ mguevara on Mar 31, 2022, 14:17

Hi @dbeltrankyl ,

I was creating an autosubmit experiment an the following message appeared:

```
...
Resolving deltas: 100% (1203/1203), done.
Checking connectivity... done.
Cloning into 'sources/monarch-DA'...
Username for 'https://earth.bsc.es': mguevara
Password for 'https://mguevara@ earth.bsc.es':
remote: Enumerating objects: 1912, done.
remote: Counting objects: 100% (25/25), done.
remote: Compressing objects: 100% (19/19), done.
remote: Total 1912 (delta 6), reused 13 (delta 6), pack-reused 1887
Receiving objects: 100% (1912/1912), 781.64 KiB | 43.00 KiB/s, done.
Resolving deltas: 100% (1154/1154), done.
Checking connectivity... done.
Cloning into 'sources/nmmb-monarch'...
Username for 'https://earth.bsc.es': mguevara
Password for 'https://mguevara@ earth.bsc.es':
remote: Enumerating objects: 29749, done.
remote: Counting objects: 100% (55/55), done.
remote: Compressing objects: 100% (27/27), done.
remote: Total 29749 (delta 28), reused 54 (delta 28), pack-reused 29694
Receiving objects: 100% (29749/29749), 192.53 MiB | 1.03 MiB/s, done.
Resolving deltas: 100% (15670/15670), done.
Checking connectivity... done.
Cloning into 'MODEL/SRC_LIBS/camp'...
Username for 'https://earth.bsc.es': mguevara
Password for 'https://mguevara@ earth.bsc.es':
remote: Enumerating objects: 36976, done.
remote: Counting objects: 100% (489/489), done.
remote: Compressing objects: 100% (192/192), done.
remote: Total 36976 (delta 349), reused 428 (delta 288), pack-reused 36487
Receiving objects: 100% (36976/36976), 164.29 MiB | 1.64 MiB/s, done.
Resolving deltas: 100% (29018/29018), done.
Checking connectivity... done.
[ERROR] Trace:
 [CRITICAL] Unhandled error: If you see this message, please report it in Autosubmit's GitLab project

```
Any idea on what could be the error?

If I put the command for creating the experiment again, the same message appear:

```
mguevara@ bscesautosubmit01:/esarchive/autosubmit/a4ic/conf> autosubmit create a4ic -np
Preparing .lock file to avoid multiple instances with same expid.

Checking configuration files...
expdef_a4ic.conf OK
platforms_a4ic.conf OK
jobs_a4ic.conf OK
wrappers OK
autosubmit_a4ic.conf OK
Configuration files OK

Using project folder: /esarchive/autosubmit/a4ic/proj
[ERROR] Trace:
 [CRITICAL] Unhandled error: If you see this message, please report it in Autosubmit's GitLab project
```
I am using the default autosubmit version `autosubmit/3.13.0-foss-2015a-Python-2.7.9``

Thanks!

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 31, 2022, 15:12

Hello @ mguevara ,

The issue is that you have in expdef.conf:
 
```
# If PROJECT_TYPE is none, no need to change
[project_files]
# Where is PROJECT CONFIGURATION file location relative to project root path
FILE_PROJECT_CONF = conf/autosubmit/proj_REGIONAL_CHEM.conf
# Where is JOBS CONFIGURATION file location relative to project root path
FILE_JOBS_CONF = conf/autosubmit/jobs_REGIONAL_CHEM.conf
# Default job scripts type in the project. type = STRING, default = bash, supported = 'bash', 'python' or 'r'
```

What is triggering this issue is that the files specified on there don't exist. Also the jobs.conf was erased so you would need to copy it again from the original experiment. 

Please, delete these paths or put a correct path

this issue has been fixed on 3.14.0b. https://earth.bsc.es/gitlab/es/autosubmit/-/issues/789

**Comment by kinow** [comment]
In GitLab by @ mguevara on Mar 31, 2022, 15:26

thanks for your answer @dbeltrankyl !
